ubuntu_dockerfile

```diff
@@ -1,237 +1,218 @@
+# limitations under the License.
+
+##########################################################################
+
+# INSTRUCTIONS:
+#
+# This Dockerfile allows for an easy installation of Oppia for Windows
+# users and a more reliable testing environment for running test scripts.
+#
+# Current known limitations:
+#   - no support for run_e2e_tests.sh
+#   - run_frontend_tests.py might not run correctly every time
+#
+# For more detailed instructions, go to the Windows page on here:
+# https://github.com/oppia/oppia/wiki/Installing-Oppia-%28Windows%29
+#
+##########################################################################
+###################### TO SET UP THE OPPIA REPOSITORY ####################
+##########################################################################
+#
+########## START DOCKER
+#
+# STEP 1
+# Download Docker Desktop for Windows.
+# Windows Enterprise and Pro: 
+#   https://hub.docker.com/editions/community/docker-ce-desktop-windows
+# Windows 10 Home: 
+#   https://docs.docker.com/toolbox/toolbox_install_windows/
+#
+# STEP 2
+# Start Docker by clicking on the Docker application (a Docker icon should
+# appear in your taskbar tray).
+#
+########## BUILD THE DOCKER IMAGE
+#
+# STEP 3
+# Build the Docker image from the Oppia root folder (note the "." at the
+# end of the command):
+#
+#   docker build -t {image_name} -f ubuntu_dockerfile .
+#      where you should replace {image_name} with whatever you want to call
+#      your Docker image (say oppia_image).
+#
+# Expect up to a 2 minute delay until the first line of output is printed.
+# Runtime: 15-25 minutes
+#
+########## BUILD A DOCKER CONTAINER BASED ON IMAGE
+#
+# STEP 4
+# Create a Docker container using the Docker image:
+#
+#   docker run -u 0 -it -p 8181:8181 --name {container_name} 
+#   -v {path_to_oppia_parent_dir}:/home {image_name}:latest bash
+#      where you should replace {container_name} with whatever you want to
+#      call your Docker container (say oppia_container),
+#      {path_to_oppia_parent_dir} with the absolute path to your oppia
+#      folder's parent directory, and {image_name} with the name of your
+#      Docker image.
+#
+# STEP 5
+# At this point, a container is built with your current oppia directory.
+# Now you should have a new terminal prompt "root@...". This is a Linux
+# terminal. Type “exit” to return to your Command Prompt.
+#
+##########################################################################
+######################## TO START THE OPPIA SERVER #######################
+##########################################################################
+#
+########## ENSURE CONTAINER IS RUNNING
+#
+# STEP 1
+# Run:
+#
+#   docker ps
+#
+# If this outputs a container, move on to step 3.
+# Note: the {container_name} in the following steps can be found under the
+# “NAMES” column of the output of “docker ps”.
+#
+# If this does not output a container, move on to step 2.

```

Comment:
You should also add if that if they see anything, then they're done.

Reply:
Oh, good point. Just added it in the new push